explore/src/main/scala/explore/config/sequence/SequenceTile.scala

```diff
@@ -96,107 +100,109 @@ object SequenceTile:
-                      GmosSouthSequenceTable(
-                        visitsOpt.collect { case ExecutionVisits.GmosSouth(visits) =>
-                          visits.toList
-                        }.orEmpty,
-                        config,
-                        snPerClass
-                      )
-                  },
-              errorRender = m =>
-                <.div(ExploreStyles.SequencesPanelError)(
-                  Message(
-                    text = m.getMessage,
-                    severity = Message.Severity.Warning,
-                    icon = Icons.ExclamationTriangle
-                  )
-                )
-            )
+    def obsEditSubscription(obsId: Observation.Id)(using StreamingClient[IO, ObservationDB]) =
+      ObsQueriesGQL.ObservationEditSubscription
+        .subscribe[IO]:
+          obsId.toObservationEditInput
 
-  private case class Title(obsExecution: Pot[Execution]) extends ReactFnProps(Title.component)
+    def executionEventAddedSubscription(obsId: Observation.Id)(using
+      StreamingClient[IO, ObservationDB]
+    ) = VisitQueriesGQL.ExecutionEventAddedSubscription.subscribe[IO](
+      ExecutionEventAddedInput(observationId = obsId.assign)
+    )
 
-  private object Title:
-    private type Props = Title
+    def queryTriggers(obsId: Observation.Id)(using StreamingClient[IO, ObservationDB]) =
+      for {
+        o <- obsEditSubscription(obsId)
+        e <- executionEventAddedSubscription(obsId)
+      } yield o.merge(e)
 
     private val component =
-      ScalaFnComponent
-        .withHooks[Props]
-        .render: props =>
-          <.span(ExploreStyles.SequenceTileTitle)(
-            props.obsExecution.orSpinner: execution =>
-              val programTimeCharge = execution.programTimeCharge.value
-
-              val executed = timeDisplay("Executed", programTimeCharge)
-
-              execution.programTimeEstimate
-                .map: plannedTime =>
-                  val total   = programTimeCharge +| plannedTime
-                  val pending = timeDisplay("Pending", plannedTime)
-                  val planned = timeDisplay("Planned", total)
-
-                  React.Fragment(
-                    HelpIcon("target/main/sequence-times.md".refined),
-                    planned,
-                    executed,
-                    pending
-                  )
-                .getOrElse(executed)
+      ScalaFnComponent[Body]: props =>
+        for {
+          ctx          <- useContext(AppContext.ctx)
+          visits       <- useStreamResourceOnMount: // Query Visits
+                            import ctx.given
+
+                            ObservationVisits[IO]
+                              .query(props.obsId)
+                              .raiseGraphQLErrors
+                              .map(_.observation.flatMap(_.execution))
+                              .attemptPot
+                              .reRunOnResourceSignals:
```

Comment:
I switched from `resetOnResourceSignals` to `reRunOnResourceSignals` to that the Pot doesn't go to `Pending` between the signal and the query results. This was causing the table to reload for each step, which wasn't very nice when getting updates from the observation execution.